cmd/scollector/main.go

```diff
@@ -245,17 +245,27 @@ func main() {
 		}
 		collect.MaxQueueLen = conf.MaxQueueLen
 	}
-	maxMemMegaBytes := uint64(500)
+	maxMemMB := uint64(500)
```

Comment:
I'm not sure I love the default of having the kill switch active. May surprise some people.